core/src/main/kotlin/fr/sncf/osrd/stdcm/infra_exploration/InfraExplorerWithEnvelope.kt

```diff
@@ -73,13 +74,15 @@ fun initInfraExplorerWithEnvelope(
     fullInfra: FullInfra,
     location: PathfindingEdgeLocationId<Block>,
     endBlocks: Collection<BlockId> = setOf(),
-    rollingStock: RollingStock
+    rollingStock: RollingStock,
+    constraints: ConstraintCombiner<BlockId, Block> = ConstraintCombiner()
```

Comment:
This *could* be a generic `EdgeToRange` instead of a `ConstraintCombiner` (though the name isn't ideal and could use an alias to `PathfindingConstraint`). In the tests, you wouldn't need to instantiate a combiner of a single constraint, you would directly input the electrification constraint.

But it's not really important, it's fine as it is